### ISSUE TITLE
Option to use themes other than powerlevel10k

### ISSUE THREAD

**Issue by mesajidiqbal** [romkatv/zsh4humans#211]
I wanted to use *[oh-my-posh](https://ohmyposh.dev/)* and its theme along with *zsh4humans*.
Although I'm able to set the setting to not install `oh-my-zsh` by commenting the line in `.zshrc`, I am not able to find any option to change the theme and use themes provided by *[oh-my-posh](https://ohmyposh.dev/)*.

Any help would be highly appriciated.

Suggestion:
Also, adding option to choose `oh-my-zsh & powerlevel10k` or `oh-my-posh` while installing `zsh4humans` will be really awesome.

**Comment by romkatv** [comment]
> I wanted to use _[oh-my-posh](https://ohmyposh.dev/)_ and its theme along with _zsh4humans_. Although I'm able to set the setting to not install `oh-my-zsh` by commenting the line in `.zshrc`, I am not able to find any option to change the theme and use themes provided by _[oh-my-posh](https://ohmyposh.dev/)_.

You aren't missing anything. zsh4humans doesn't have an option to replace powerlevel10k with a different theme.

> Suggestion: Also, adding option to choose `oh-my-zsh & powerlevel10k` or `oh-my-posh` while installing `zsh4humans` will be really awesome.

This isn't something I'll be implementing or merging a PR for.

Closing as Working as Intended / Won't Fix.

**Comment by mesajidiqbal** [comment]
Just adding my workaround in case someone needs it:

- Install zsh4humans
- Install oh-my-posh
- Go to $Z4H folder (check value by echo command)
- Delete everything from powerlevel10k/powerlevel10k.zsh-theme file
- Restart the terminal

And thanks @romkatv for [this](https://github.com/romkatv/zsh4humans) great and easy configuration

**Comment by romkatv** [comment]
I'm curious why you would want that. As far as I know (from using oh-my-posh and reading its code), powerlevel10k is strictly better by a large margin.

Note that your workaround breaks functionality in zsh4humans. What exactly it breaks is of course unspecified.

**Comment by mesajidiqbal** [comment]
I have been using zsh4humans for many years.
But recently, when I came across oh-my-posh, I liked the `jandedobbeleer` theme.
I just wanted to try it out for a few days (I know powerlevel10k color schemes can also be altered, but I'm not very good at it), so I installed oh-my-posh.
While I don't want to lose any of the other functionality provided by zsh4humans (since I've been using it for a long time, I'm quite accustomed to it)
Therefore, I was looking for a workaround.
